app/controllers/api/api.js

```diff
@@ -18,4 +21,82 @@ function authenticatedUser(req, res, next) {
 
 router.get('/secret', authenticatedUser, function (req, res, next) {
   res.json({message: "secret"});
-});
\ No newline at end of file
+});
+
+// INDEX
+router.get('/api/helpz', function (req, res){
+  Service.find({}, function (err, services) {
+    if (err) {
+      res.json({message: 'Could not find any Services b/c: ' + err});
```

Comment:
For your error responses, perhaps it would be useful to send a success key as well:

```js
res.status(422).json({success: false, message: err})
```

That way you can always check for `success` in your client-side :)